plugins/woocommerce/client/admin/client/settings-payments/components/buttons/enable-gateway-button.tsx

```diff
@@ -126,6 +131,7 @@ export const EnableGatewayButton = ( {
 						}
 					}
 				}
+				// if no redirect occured, the data needs to be refreshed.
```

Comment:
All phrases should start with uppercase.

Reply:
thanks! addressed in https://github.com/woocommerce/woocommerce/pull/54263/commits/1cb4f8eb9290355d38f8cefebb9784d70bdf6450